Tests/BrokerServiceTests/UnitTest1.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace BrokerServiceTests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1()
+        {
+            Assert.Pass();
```

Comment:
I am not sure that this template is good. We don't want _UnitTest1.cs_ and `poblic void Test1` in develop branch.

Reply:
> I am not sure that this template is good. We don't want _UnitTest1.cs_ and `poblic void Test1` in develop branch.

This is just an empty project. Naturally, in the future the name will be correct, but so far the testing logic has not been developed so it makes no sense to come up with a name just like that

Reply:
Agree with Maksim